README.md

```diff
@@ -4,6 +4,12 @@
 
 <h4 align="center">Perun State Channels Framework - Ethereum Backend Smart Contracts</h4>
 
+<p align="center">
+  <a href="https://codecov.io/gh/hyperledger-labs/perun-eth-contracts"><img src="https://codecov.io/gh/hyperledger-labs/perun-eth-contracts/branch/master/graph/badge.svg?token=QXZH8MKQG5" alt="Codecov"></a>
```

Comment:
Good idea to add the batch! The badge is currently showing "unknown" on https://github.com/perun-network/perun-eth-contracts/tree/fix/mythril-ci. Can you check?

Reply:
Yes since the default branch is `master` and it did not yet get merged into master.